introduction.md

```diff
@@ -26,7 +26,7 @@ Asking good questions can be hard, generally it's best to say what you are tryin
 
 ## Cross posting
 
-Please try to refrain from cross-posting in multiple channels. If you have asked your question in a channel and have waited a while you can move to a more [#general](https://pythondev.slack.com/messages/C07EFBK3R) channel. In order to cross-post, find your chat message and [share the message](https://get.slack.help/hc/en-us/articles/203274767-Share-messages-in-Slack) in the other channel.
+Please try to refrain from cross-posting in multiple channels. If you have asked your question in a channel and have waited a while you can move to a more [#random](https://pythondev.slack.com/messages/C07EFBK3R) channel. In order to cross-post, find your chat message and [share the message](https://get.slack.help/hc/en-us/articles/203274767-Share-messages-in-Slack) in the other channel.
```

Comment:
I don't think we should link to a channel here. The channel for question should be #help. Leaving it at 
`If you have asked your question in a channel and have waited a while you can move to a more general channel.` without link is best IMO.

Reply:
Agreed with this, maybe even say "a more broad channel" - to avoid any confusion about if we are talking about a specific channel.